src/routes/muzzle-route.ts

```diff
@@ -46,7 +55,6 @@ muzzleRoutes.post("/muzzle", (req: Request, res: Response) => {
   const request: ISlashCommandRequest = req.body;
   const userId: string = getUserId(request.text);
   const userName: string = getUserName(request.text);
-  console.log(request);
   try {
```

Comment:
This block is a little weird to me... Why is there a try catch here inside of this synchronous function?

Reply:
If we are worried about accessing a property on an undefined value perhaps lodash get should be used instead

Reply:
The reason this is because it is possible that `addUserToMuzzled` returns an error in which case i want to send that error message back to the user, and only the message, not the full error object because the message is the only thing that the user is concerned with. Things like `You muzzled too much in the past hour`, or `That person is already muzzled.` Maybe there is a cleaner way to achieve the same functionality?

Reply:
I try and void the use of try catch basically altogether (as well as in async funcs) but maybe thats just a style choice (I have heard that JS is not optimized to handle try catch blocks). I prefer to handle errors explicitly exactly where they occur instead of having a big dumpster catch block although in this exact case it looks like the error can only happen in one place at the invocation of `addUserToMuzzled()` so it doesnt really matter


You could do something like this:
```
export function addUserToMuzzled(
  toMuzzle: string,
  friendlyMuzzle: string,
  requestor: string
) {
  return new Promise((resolve, reject) => {
    const timeToMuzzle = Math.floor(
      Math.random() * (180000 - 30000 + 1) + 30000
    );
    const minutes = Math.floor(timeToMuzzle / 60000);
    const seconds = ((timeToMuzzle % 60000) / 1000).toFixed(0);
    if (muzzled.includes(toMuzzle)) {
      console.error(
        `${requestor} attempted to muzzle ${toMuzzle} but ${toMuzzle} is already muzzled.`
      );
      return reject(`${friendlyMuzzle} is already muzzled!`);
    } else if (muzzled.includes(requestor)) {
      console.error(
        `User: ${requestor} attempted to muzzle ${toMuzzle} but failed because requestor: ${requestor} is currently muzzled`
      );
      return reject(`You can't muzzle someone if you are already muzzled!`);
    } else {
      muzzled.push(toMuzzle);
      console.log(
        `${friendlyMuzzle} is now muzzled for ${timeToMuzzle} milliseconds`
      );
      setTimeout(() => removeMuzzle(toMuzzle), timeToMuzzle);
      return resolve(
        `Succesfully muzzled ${friendlyMuzzle} for ${
          +seconds === 60
            ? minutes + 1 + "m00s"
            : minutes + "m" + (+seconds < 10 ? "0" : "") + seconds + "s"
        } minutes`
      );
    }
  });
}
``` 

```
muzzleRoutes.post("/muzzle", (req: Request, res: Response) => {
  const request: ISlashCommandRequest = req.body;
  const userId: string = getUserId(request.text);
  const userName: string = getUserName(request.text);
  const results = addUserToMuzzled(userId, userName, request.user_id).catch(e =>
    res.send(e.message)
  );
  if (results) res.send(results);
});
```

Reply:
I'm gunna merge this one and do a separate PR for the stuff you mentioned here because I like this better than what I was doing